src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -207,14 +178,13 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     }
 }
 
-
 //! Clears the toolbox.
 void medRegistrationSelectorToolBox::clear(void)
```

Comment:
Yes, and QT creator will find the code that uses the slot in a connection.
Another victory for QT Creator...